src/library/scala/io/Source.scala

```diff
@@ -167,6 +167,11 @@ object Source {
 
   def fromInputStream(is: InputStream)(implicit codec: Codec): BufferedSource =
     createBufferedSource(is, reset = () => fromInputStream(is)(codec), close = () => is.close())(codec)
+
+  def fromClassPath(resource: String, classLdr: ClassLoader = ClassLoader.getSystemClassLoader)(implicit codec: Codec): BufferedSource = {
+    fromInputStream(classLdr.getResourceAsStream(resource))(codec)
+  }
```

Comment:
I wouldn't put that in braces, but I didn't want to cause friction here, since we hadn't been as prompt with our review as I'd have liked.